src/tests.adoc

```diff
@@ -0,0 +1,33 @@
+= Tests
+
+== Global Range
+
+== Gradient
+
+[stem]
+++++
+X_i = \left| V_i - \frac{V_{i+1} + V_{i-1}}{2} \right|
+++++
+
+== Spike
+
+[stem]
+++++
+X_i = \left| V_i - \frac{V_{i+1} + V_{i-1}}{2} \right| - \left| \frac{V_{i+1} - V_{i-1}}{2} \right|
+++++
+
+== Digit Rollover
+
+[stem]
+++++
+X_i = V_i - V_{i-1}
+++++
+
+== Spike - Median
+
+Proposed at <<BGC_nitrate>>
+
+[stem]
+++++
+X_i = \left| V_2 - median(V_0, V_1, V_2, V_3, V_4, V_5) \right|
```

Comment:
This notation is inconsistent with the other spike test above. This should be something like:

X_i = \left| V_i - median(V_i-2, V_i-1, V_i, V_i+1, V_i+2, V_i+3) \right|

It also seems odd to take a median of 6 values. Wouldn't an odd number centered on the index *i* that the QC refers to make more sense?

Reply:
The spike test defined in the cited document [BGC-Argo quality control manual for nitrate concentration](https://archimer.ifremer.fr/doc/00732/84370/89380.pdf) defines a median filter with 5 values, centered on the value being tested.

Reply:
@callumrollo, thanks! My bad.